src/service/ServiceManager.java

```diff
@@ -89,6 +96,15 @@
 	private String milestonesETag = null;
 	private String issueCheckTime = null;
 	
+	private final ScheduledExecutorService pollExecutor = Executors.newScheduledThreadPool(1);
+	private ScheduledFuture<?> pollHandler;
+	private long pollInterval = 60000; //time between polls in ms
+	private int stopwatchInterval = 1000;
+	private int timeRemaining = 60;
+	private static final int SECS = 60;
```

Comment:
Maybe you could move `SECS` to the top and have the other variables depend on it, i.e. `pollInterval = SECS * 1000`, etc. Also it could be named `TIME_BETWEEN_POLLS` (from the comment) or something.